index.php

```diff
@@ -97,6 +97,8 @@ function admin_init() {
 		\wp_enqueue_script( 'jquery-ui-datepicker' );
 		\wp_enqueue_script( 'acf-input' );
 		\wp_enqueue_style( 'acf-input' );
+		\wp_enqueue_style( 'acf-datepicker', acf_get_dir( 'assets/inc/datepicker/jquery-ui.min.css' ), '', '1.11.4' );
```

Comment:
Dependencies are an array and not a string, even when there is only one or none. The default is `array()`, use that instead of `''`.